### DISCUSSION TITLE
[v4] button cursor pointer

### DISCUSSION THREAD

**Discussion by cuquo** [tailwindlabs/tailwindcss#15203] [Help]
in v3 buttons have cursor-pointer and in v4 is set to default, not sure if is a known issue

**Comment by wongjn** [comment]
I believe it is on purpose. If we look at #14061, (which talks about some rule overriding `cursor` which is no longer needed), we can see it references #8961 and #8962. I believe these could be the reasons why buttons do not have `cursor: pointer` in preflight in v4.

**Comment by julie598doyle** [comment]
Hello,
It seems like a known issue in Tailwind CSS v4. In v3, buttons had cursor-pointer by default, but this was changed in v4 to improve accessibility. Buttons now use the default cursor to indicate they are not links.

Best Regards,
Julie Doyle

**Comment by jlemonz** [comment]
yes some issue. I was confused because this is not in the upgrade docs

**Comment by imakeinternet** [comment]
This change should be included in the upgrade documentation. I spent more time than I’d like to admit trying to figure out why upgrading broke all my button cursors. Thankfully, I eventually stumbled upon #15773, and it all made sense. At least now I know I'm not crazy.

**Reply by adamwathan** [reply]
It’s in there but maybe easy to miss:

https://tailwindcss.com/docs/upgrade-guide#buttons-use-the-default-cursor

**Reply by imakeinternet** [reply]
@adamwathan Oh my goodness, how did I miss that? Now I feel completely silly. Thank you for bringing that to my attention.
